src/main/java/edu/harvard/iq/dataverse/export/SchemaDotOrgExporter.java

```diff
@@ -92,6 +102,266 @@ public void exportDataset(DatasetVersion version, JsonObject json, OutputStream
         }
     }
 
+
+    // The export subsystem assumes there is only
```

Comment:
This is the problem I was talking about in earlier comments - ExportService doesn't handle per-version exporting and we need that for schema.org for the page header (and will want it for other reasons soon - see #9164). In any case - once that's fixed, this comment won't be true anymore.